tgui/packages/tgui/interfaces/BotHonk.js

```diff
@@ -0,0 +1,15 @@
+import { useBackend } from '../backend';
+import { Button, LabeledList, Box, Section} from '../components';
+import { Window } from '../layouts';
+import { BotStatus } from './common/BotStatus';
+
+export const BotHonk = (props, context) => {
+  const { act, data } = useBackend(context);
+  return (
+    <Window resizable>
+      <Window.Content scrollable>
+        <BotStatus />
+      </Window.Content>
+    </Window>
+  );
+};
```

Comment:
redundunt also :D Just use bot ui. That's it

Reply:
ok I do need help on this, wdym by "bot ui"? what am I missing here?

Reply:
Let me check bot ui fast and I will be back

Reply:
Yep. UI is indeed needed. I am mistaken

Reply:
> Yep. UI is indeed needed. I am mistaken

resolving this one aight?

Reply:
yep